project/Dependencies.scala

```diff
@@ -42,6 +42,7 @@ object Dependencies {
   val jodaConvert = "org.joda" % "joda-convert" % "1.2"
   val jSoup = "org.jsoup" % "jsoup" % "1.8.3"
   val liftJson = "net.liftweb" %% "lift-json" % "2.6-RC2"
+  val macwire = "com.softwaremill.macwire" %% "macros" % "2.2.3" % "provided"
```

Comment:
Technically I don't _need_ it, but it does bring automatic wiring. In the case you selected it's straightforward to create the `JobScheduler`.

It gets a lot more messier when you need to inject components like `WsClient`, `ExecutionContext` and such.
A good example is `AdminHttpErrorHandler` or `AdminLifecycle`, or even the `Router`
modifying the route file (even the order of lines) can modify the order of the parameters in the constructor, or add new parameters, which will trigger a compilation error (that won't be caught by Intellij) whereas macwire is handling that wiring for you at compile time.

So do we need it? no, is it nice? in my opinion yes, but I'm open to hear more about why we shouldn't use it